new_platforms/nuget/build/native/openenclave.targets

```diff
@@ -0,0 +1,126 @@
+        <ClCompile>
+            <!-- SGX Hardware and Simulation --> 
+            <PreprocessorDefinitions Condition="'$(Configuration)' == 'SGX-Debug' Or '$(Configuration)' == 'SGX-Simulation-Debug'">
+                OE_USE_SGX;%(PreprocessorDefinitions)
+            </PreprocessorDefinitions>
+
+            <!-- ARM TrustZone Hardware --> 
+            <PreprocessorDefinitions Condition="'$(Configuration)' == 'ARMTZ-Debug'">
+                OE_USE_OPTEE;%(PreprocessorDefinitions)
+            </PreprocessorDefinitions>
+
+            <!-- ARM TrustZone Simulation --> 
+            <PreprocessorDefinitions Condition="'$(Configuration)' == 'ARMTZ-Simulation-Debug'">
+                OE_SIMULATE_OPTEE;%(PreprocessorDefinitions)
+            </PreprocessorDefinitions>
+        </ClCompile>
+    </ItemDefinitionGroup>
+
+    <!-- Linking Rules for Enclaves -->
+    <ItemDefinitionGroup Condition="'$(OEType)' == 'Enclave'">
+        <Link>
+            <!-- All, except ARM TrustZone Simulation -->
+            <IgnoreAllDefaultLibraries Condition="'$(Configuration)' != 'ARMTZ-Simulation-Debug'">
+                true
+            </IgnoreAllDefaultLibraries>
+
+            <!-- x86-SGX-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(SgxHwX86Dbg)\oeenclave.lib;$(SgxHwX86Dbg)\oesocket_enc.lib;$(SgxHwX86Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x64-SGX-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Debug' And '$(Platform)' == 'x64'">
+                $(SgxHwX64Dbg)\oeenclave.lib;$(SgxHwX64Dbg)\oesocket_enc.lib;$(SgxHwX64Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x86-SGX-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Simulation-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(SgxSimX86Dbg)\oeenclave.lib;$(SgxSimX86Dbg)\oesocket_enc.lib;$(SgxSimX86Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x64-SGX-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Simulation-Debug' And '$(Platform)' == 'x64'">
+                $(SgxSimX64Dbg)\oeenclave.lib;$(SgxSimX64Dbg)\oesocket_enc.lib;$(SgxSimX64Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x86-ARMTZ-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'ARMTZ-Simulation-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(TzSimX86Dbg)\oeenclave.lib;$(TzSimX86Dbg)\oesocket_enc.lib;$(TzSimX86Dbg)\oestdio_enc.lib;$(TzSimX86Dbg)\oeenclave_opteesim.lib;%(AdditionalDependencies)
```

Comment:
I think oeenclave_opteesim.lib also depends on
kernel32.lib;vcruntime.lib;ucrtd.lib;advapi32.lib

Reply:
Those should be pulled via `%(AdditionalDependencies)` as far as I know.